src/static/styles/App.scss

```diff
@@ -309,11 +360,17 @@ hr {
 }
 
 .bordered-background1 {
-	border: 4.775rem solid #262867;
+	border: 3rem solid #262867;
+	@include md{
+		border: 4.775rem solid #262867;
```

Comment:
if possible can you encode all colors as variables at the top?

Reply:
Yes. I can implement these changes when implementing the new mobile design. However, I will merge this for now since these issues are urgent.